.zenodo.json

```diff
@@ -1,40 +1,89 @@
 {
-  "creators": [
-    {
-      "name": "DuPre, Elizabeth",
-      "affiliation": "Montreal Neurological Institute, McGill University",
-      "orcid": "0000-0003-1358-196X"
-    },
-    {
-      "name": "Salo, Taylor",
-      "affiliation": "Florida International University",
-      "orcid": "0000-0001-9813-3167"
-    },
-    {
-      "name": "Markello, Ross",
-      "affiliation": "Montreal Neurological Institute, McGill University",
-      "orcid": "0000-0003-1057-1336"
-    },
-    {
-      "name": "Kundu, Prantik",
-      "affiliation": "Mount Sinai Hospital",
-      "orcid": "0000-0001-9367-3068"
-    },
-    {
-      "name": "Whitaker, Kirstie",
-      "affiliation": "Alan Turing Institute; Cambridge University",
-      "orcid": "0000-0001-8498-4059"
-    },
-    {
-      "name": "Handwerker, Dan",
-      "affiliation": "National Institutes of Health",
-      "orcid": "0000-0001-7261-4042"
-    }
-  ],
-  "keywords": [
-    "neuroimaging",
-    "fMRI"
-  ],
-  "license": "LGPL-2.1",
-  "upload_type": "software"
+    "creators": [
+        {
+            "name": "Caballero-Gaudes, César",
```

Comment:
Are we putting "The tedana Community" first ?

Reply:
I... don't know. Since the community is reflected in the Zenodo file, do we need a collective name? Sorry, I honestly don't remember where we ended up on this.

Reply:
I'm thinking back to what we did on this year's poster.. I think the benefit of having a collective author first is it captures anyone not in the Zenodo, and it also helps to clarify that this is a community project when cited. That is, the citation would be (The tedana Community, 20XX) rather than (Author, 20XX).

Reply:
I like that. It avoids the false first authorship of the person with the earliest last name. I will add it. Just the name, no other metadata, right?

Reply:
I do believe that we agreed on tedana community being first on posters and on zenodo, but not necessarily for papers, based on previous dev calls.

Reply:
I think that's the only metadata we have ! :smile_cat: It seems there's still not a great way to validate these files before submitting, though (see: https://github.com/zenodo/zenodo/issues/1606). So I suppose we can try it and adjust from there ?